documentation/asciidoc/computers/camera/rpicam_apps_multicam.adoc

```diff
@@ -1,12 +1,10 @@
-=== Using multiple cameras 
+=== Use multiple cameras
 
-Basic support for multiple cameras is available within `rpicam-apps`. Multiple cameras may be attached to a Raspberry Pi in the following ways:
+`rpicam-apps` has basic support for multiple cameras. You can attach multiple cameras to a Raspberry Pi in the following ways:
 
-* Two cameras connected directly to a Raspberry Pi Compute Module board, see the xref:../computers/compute-module.adoc#attach-a-raspberry-pi-camera-module[Compute Module documentation] for further details.
-* Two or more cameras attached to a non-compute Raspberry Pi board using a Video Mux board, like https://www.arducam.com/product/multi-camera-v2-1-adapter-raspberry-pi/[this 3rd party product].
+* For Raspberry Pi Compute Modules, you can connect two cameras directly to a Raspberry Pi Compute Module I/O board. See the xref:../computers/compute-module.adoc#attach-a-raspberry-pi-camera-module[Compute Module documentation] for further details. With this method, you can _use both cameras simultaneously_.
+* For Raspberry Pi devices other than Compute Modules, you can attach two or more cameras with a Video Mux board such as https://www.arducam.com/product/multi-camera-v2-1-adapter-raspberry-pi/[this third-party product]. Since both cameras are attached to a single Unicam port, _only one camera may be used at a time_.
 
-In the latter case, only one camera may be used at a time, since both cameras are attached to a single Unicam port. For the former, both cameras can run simultaneously.
-
-To list all the cameras available on your platform, use the `--list-cameras` command line option. To choose which camera to use, use the `--camera <index>` option, and provide the index value of the requested camera.
+To list all the cameras available on your platform, use the xref:camera_software.adoc#list-cameras[`list-cameras`] option. To choose which camera to use, pass the xref:camera_software.adoc#camera[`camera`] option the index value of the requested camera.
```

Comment:
"To choose which camera to use, pass the xref:camera_software.adoc#camera[`camera`] option the index value of the requested camera." -> "To choose which camera to use, pass the camera index to the xref:camera_software.adoc#camera[`camera`] option."
?